.travis.yml

```diff
@@ -25,6 +25,9 @@ cache:
 install:
   - pip install tox
 
+before_script:
+  - psql -c 'create database spotify;' -U postgres
```

Comment:
It would be possible by doing
```bash
[ $TOX_ENV = 'nonhdfs' ] && psql -c 'create database spotify;' -U postgres
```

However I'm not sure of the advantage of doing so. Postgres is installed and running anyway. Creating a DB is not really expensive.